test/puppeteer_utils.test.js

```diff
@@ -375,8 +376,10 @@ describe('Apify.utils.puppeteer', () => {
 
             await Apify.utils.puppeteer.saveSnapshot(page, { key: 'TEST', keyValueStoreName: 'TEST-STORE', screenshotQuality: 60 });
 
-            expect(stub.calledWithExactly('TEST.jpg', screenshot, { contentType: 'image/jpeg' })).toBe(true);
-            expect(stub.calledWithExactly('TEST.html', contentHTML, { contentType: 'text/html' })).toBe(true);
+            const screenshotName = utils.isAtHome() ? 'TEST.jpg' : 'TEST';
+            expect(stub.calledWithExactly(screenshotName, screenshot, { contentType: 'image/jpeg' })).toBe(true);
+            const htmlName = utils.isAtHome() ? 'TEST.html' : 'TEST';
+            expect(stub.calledWithExactly(htmlName, contentHTML, { contentType: 'text/html' })).toBe(true);
```

Comment:
This does not really test both situations, because the test will never run on Apify platform, so `isAtHome()` will always be `false` You need to add a test case for that and either mock the function to return `true` or set `APIFY_IS_AT_HOME=1` env var (i think it's that name, but better check)